client/state/jetpack-connect/actions.js

```diff
@@ -72,4 +80,52 @@ export default {
 	goToPluginInstall( url ) {
 		window.location = url + installURL;
 	},
+	createAccount( userData ) {
+		return ( dispatch ) => {
+			dispatch( {
```

Comment:
maybe we should add some tracking both tot the create account action & the response handling (especially if there's an error)